easycv/apis/export.py

```diff
@@ -526,115 +585,18 @@ def __call__(
-    """Model export wrapper that supports end-to-end export of pre-processing and post-processing.
-    We support some built-in preprocessing and postprocessing functions.
-    If the requirements are not met, you can customize the preprocessing and postprocessing functions.
-    The custom functions must support satisfy requirements of `torch.jit.script`,
-    please refer to: https://pytorch.org/docs/stable/jit_language_reference_v2.html
-
-    Args:
-        model (torch.nn.Module):  `torch.nn.Module` that will be run with `example_inputs`.
-            `model` arguments and return values must be tensors or (possibly nested) tuples
-            that contain tensors. When a module is passed `torch.jit.trace`, only the
-            ``forward_export`` method is run and traced (see :func:`torch.jit.trace
-            <torch.jit.trace_module>` for details).
-        example_inputs (tuple or torch.Tensor):  A tuple of example inputs that
-            will be passed to the function while tracing. The resulting trace
-            can be run with inputs of different types and shapes assuming the
-            traced operations support those types and shapes. `example_inputs`
-            may also be a single Tensor in which case it is automatically
-            wrapped in a tuple.
-        preprocess_fn (callable or None): A Python function for processing example_input.
-            If there is only one return value, it will be passed to `model.forward_export`.
-            If there are multiple return values, the first return value will be passed to `model.forward_export`,
-            and the remaining return values ​​will be passed to `postprocess_fn`.
-        postprocess_fn (callable or None): A Python function for processing the output value of the model.
-            If `preprocess_fn` has multiple outputs, the output value of `preprocess_fn`
-            will also be passed to `postprocess_fn`. For details, please refer to: `preprocess_fn`.
-        trace_model (bool): If True, before exporting the end-to-end model,
-            `torch.jit.trace` will be used to export the `model` first.
-            Traceing an ``nn.Module`` by default will compile the ``forward_export`` method and recursively.
-
-    Examples:
-        import torch
-
-        batch_size = 1
-        example_inputs = 255 * torch.rand((batch_size, 3, 640, 640), device='cuda')
-        end2end_model = End2endModelExportWrapper(
-            model,
-            example_inputs,
-            preprocess_fn=PreProcess(target_size=(640, 640)),  # `PreProcess` refer to ev_torch.apis.export.PreProcess
-            postprocess_fn=DetPostProcess()  # `DetPostProcess` refer to ev_torch.apis.export.DetPostProcess
-            trace_model=True)
-
-        model_script = torch.jit.script(end2end_model)
-        with io.open('/tmp/model.jit', 'wb') as f:
-            torch.jit.save(model_script, f)
-    """
+class ModelExportWrapper(torch.nn.Module):
```

Comment:
why remove End2endModelExportWrapper and preprocess_fn, postprocess_fn

Reply:
we will discuss it.